CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 - Active Subscription
   - Switch endpoint used to display "App Subscriptions"
-- Technical User
+- Technical User Details
+  - Reset credential
```

Comment:
Could you please change to "Enable credential reset"?
I think that leaves less room for interpretation.